.github/workflows/test-race.yml

```diff
@@ -101,6 +101,22 @@ jobs:
             **/**.go
             go.mod
             go.sum
+      - name: Get data from build cache
+        uses: actions/cache@v2
+        with:
+           # In order:
+           # * Module download cache
+           # * Build cache (Linux)
+           # * Build cache (Mac)
+           # * Build cache (Windows)
+          path: |
+             ~/go/pkg/mod
+             ~/.cache/go-build
+             ~/Library/Caches/go-build
+             ~\AppData\Local\go-build
+          key: ${{ runner.os }}-go-${{ matrix.go-version }}-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+             ${{ runner.os }}-go-${{ matrix.go-version }}-
```

Comment:
Perhaps namespace this to `go-race-` so it makes a separate cache from the non-race tests?

I believe this matters for the test cache